Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
 
 == RMM Flash-Bang Distraction and Clearing Device $50 ==
 
-Damage: D10 burn damage to exact spot of impact
+Damage: D10 fire damage to exact spot of impact
 Ranged Miss Chance (to land where intended): 6 out to 10+[DEX or STR]*2 meters
 Armor Piercing Level: 0
 Reflex Modifier: -10
 Reflex Save: 110
 
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding 

```

Comment:
What kind of check?

Reply:
Blocker: ya, we really need to know what kind of check this is.

Reply:
That's a discussion issue then. The old stuff is nonfunctional enough that there are multiple ways to fix it and I'd rather not unilaterally pick one.

Reply:
Spotting or awareness seems most likely to me.

Reply:
Making it an Awareness save or a Perception check both work for me.